src/App.tsx

```diff
@@ -1,15 +1,39 @@
-import React from 'react';
+import React, { useContext, useEffect, useState } from 'react';
 import 'bulma/bulma.sass';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
 
-import classNames from 'classnames';
-import { PostsList } from './components/PostsList';
-import { PostDetails } from './components/PostDetails';
-import { UserSelector } from './components/UserSelector';
 import { Loader } from './components/Loader';
+import { Sidebar } from './components/Sidebar';
+import { UserSelector } from './components/UserSelector';
+import { PostsList } from './components/PostsList';
+import { PostContext } from './context/PostContext';
```

Comment:
would be better to create `index.ts` file in each of the component folder
`components/ComponentName1/index.ts`
```
export * from './ComponentName1';
```

then, create `index.ts` file in the `components` folder like

`components/index.ts`

```
export * from './ComponentName1';
export * from './ComponentName2';
export * from './ComponentName3';
```

usage

`App.tsx`
```
import { ComponentName1, ComponentName2, ComponentName3 } from './components'
```